### ISSUE TITLE
PDB remote server: retry upon failure

### ISSUE THREAD

**Issue by sad-dev** [NationalSecurityAgency/ghidra#4324]
**Is your feature request related to a problem? Please describe.**
During autoanalysis, Ghidra can download PDB files from remote symbol servers. Large files or flaky connections can cause this download to fail. Ghidra then proceeds with autoanalysis without the PDB. Having to cancel autoanalysis and reload the program is a mild annoyance.

**Describe the solution you'd like**
An option to configure the number of attempts in downloading a PDB file.

**Comment by dev747368** [comment]
In general I know what you are describing, but any log files / stack traces that show a particular incident would be appreciated.

**Comment by sad-dev** [comment]
```
ERROR	(PdbPlugin) Error getting symbol file java.io.IOException: closed
			at java.net.http/jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream.current(ResponseSubscribers.java:368)
			at java.net.http/jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream.read(ResponseSubscribers.java:403)
			at java.base/java.io.InputStream.read(InputStream.java:205)
			at utilities.util.FileUtilities.copyStreamToStream(FileUtilities.java:662)
			at utilities.util.FileUtilities.copyStreamToFile(FileUtilities.java:617)
			at pdb.symbolserver.LocalSymbolStore.putStream(LocalSymbolStore.java:350)
			at pdb.symbolserver.SymbolServerService.ensureLocalUncompressedFile(SymbolServerService.java:245)
			at pdb.symbolserver.SymbolServerService.getSymbolFile(SymbolServerService.java:201)
			at pdb.PdbPlugin.findPdb(PdbPlugin.java:212)
			at ghidra.app.plugin.core.analysis.PdbAnalyzerCommon.findPdb(PdbAnalyzerCommon.java:146)
			at ghidra.app.plugin.core.analysis.PdbUniversalAnalyzer.added(PdbUniversalAnalyzer.java:166)
			at ghidra.app.plugin.core.analysis.AnalysisScheduler.runAnalyzer(AnalysisScheduler.java:186)
			at ghidra.app.plugin.core.analysis.AnalysisTask.applyTo(AnalysisTask.java:39)
			at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisTaskWrapper.run(AutoAnalysisManager.java:688)
			at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:788)
			at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:667)
			at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:632)
			at ghidra.app.plugin.core.analysis.AnalysisBackgroundCommand.applyTo(AnalysisBackgroundCommand.java:58)
			at ghidra.framework.plugintool.mgr.BackgroundCommandTask.run(BackgroundCommandTask.java:102)
			at ghidra.framework.plugintool.mgr.ToolTaskManager.run(ToolTaskManager.java:336)
			at java.base/java.lang.Thread.run(Thread.java:829)
			Caused by: java.io.IOException: fixed content-length: 3788800, bytes received: 1916226
			at java.net.http/jdk.internal.net.http.common.Utils.wrapWithExtraDetail(Utils.java:330)
			at java.net.http/jdk.internal.net.http.Http1Response$BodyReader.onReadError(Http1Response.java:758)
			at java.net.http/jdk.internal.net.http.Http1AsyncReceiver.checkForErrors(Http1AsyncReceiver.java:297)
			at java.net.http/jdk.internal.net.http.Http1AsyncReceiver.flush(Http1AsyncReceiver.java:263)
			at java.net.http/jdk.internal.net.http.common.SequentialScheduler$SynchronizedRestartableTask.run(SequentialScheduler.java:175)
			at java.net.http/jdk.internal.net.http.common.SequentialScheduler$CompleteRestartableTask.run(SequentialScheduler.java:147)
			at java.net.http/jdk.internal.net.http.common.SequentialScheduler$SchedulableTask.run(SequentialScheduler.java:198)
			at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
			at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
			... 1 more
			Caused by: java.io.IOException: connection closed locally
			at java.net.http/jdk.internal.net.http.SocketTube.signalClosed(SocketTube.java:157)
			at java.net.http/jdk.internal.net.http.PlainHttpConnection.close(PlainHttpConnection.java:277)
			at java.net.http/jdk.internal.net.http.AsyncSSLConnection.close(AsyncSSLConnection.java:111)
			at java.net.http/jdk.internal.net.http.Http1Exchange.cancelImpl(Http1Exchange.java:490)
			at java.net.http/jdk.internal.net.http.Http1Exchange.cancel(Http1Exchange.java:426)
			at java.net.http/jdk.internal.net.http.Exchange.cancel(Exchange.java:228)
			at java.net.http/jdk.internal.net.http.MultiExchange.cancel(MultiExchange.java:240)
			at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:71)
			at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1248)
			at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:877)
```
